modules/tinymce/changelog.txt

```diff
@@ -1,36 +1,36 @@
-    Added Oxide variables for styling the select element and headings in dialog content #TINY-6070
-    Added icons for `table` column and row cut, copy and paste toolbar buttons #TINY-6062
+    Added keyboard navigation support to menus and toolbars when the editor is in a ShadowRoot #TINY-6152
+    Added the ability for menus to be clicked when the editor is in an open shadow root #TINY-6091
+    Added the `Editor.ui.styleSheetLoader` API for loading stylesheets within the Document or ShadowRoot containing the editor UI #TINY-6089
+    Added the `StyleSheetLoader` module to the public API #TINY-6100
+    Added Oxide variables for styling the `select` element and headings in dialog content #TINY-6070
+    Added icons for `table` column and row cut, copy, and paste toolbar buttons #TINY-6062
     Added all `table` menu items to the UI registry, so they can be used by name in other menus #TINY-4866
     Added new `mceTableApplyCellStyle` command to the `table` plugin #TINY-6004
-    Added new `table` cut, copy and paste column editor commands and menu items #TINY-6006
-    Added additional font related Oxide variables for secondary buttons allowing more specific styling. #TINY-6061
+    Added new `table` cut, copy, and paste column editor commands and menu items #TINY-6006
+    Added font related Oxide variables for secondary buttons, allowing for custom styling #TINY-6061
     Added new `table_header_type` setting to control how table header rows are structured #TINY-6007
     Added new `table_sizing_mode` setting to replace the `table_responsive_width` setting, which has now been deprecated #TINY-6051
     Added new `mceTableSizingMode` command for changing the sizing mode of a table #TINY-6000
-    Changed `advlist` toolbar buttons to only show a dropdown list if there's more than one option #TINY-3194
+    Changed `advlist` toolbar buttons to only show a dropdown list if there is more than one option #TINY-3194
     Changed `mceInsertTable` command and `insertTable` API method to take optional header rows and columns arguments #TINY-6012
-    Changed stylesheet loading, so that UI skin stylesheets now to load in a ShadowRoot, if the editor is in a ShadowRoot. #TINY-6089
-    Changed the DOM location of menus so that they display correctly when the editor is in a ShadowRoot. #TINY-6093
+    Changed stylesheet loading, so that UI skin stylesheets can load in a ShadowRoot if required #TINY-6089
+    Changed the DOM location of menus so that they display correctly when the editor is in a ShadowRoot #TINY-6093
     Changed the table plugin to correctly detect all valid header row structures #TINY-6007
-    Fixed tables with no defined width being converted to a `fixed` width table when modifying the table #TINY-6051
-    Fixed the `autosave` isEmpty API incorrectly detecting non-empty content as empty #TINY-5953
+    Fixed tables with no defined width not being converted to a `fixed` width table when modifying the table #TINY-6051
```

Comment:
Was mentioned out of GitHub, but thought I'd flag that this doesn't have the same meaning

Reply:
Fixed in https://github.com/tinymce/tinymce/pull/5823/commits/0a676ffd5326bc5da6bd1e1d703279adf725d194